tests/wpt/mozilla/tests/mozilla/referrer-policy/generic/subresource/link.py

```diff
@@ -0,0 +1,27 @@
+import os, json, sys, urllib2, urlparse
+sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
+
+import mozresource; subresource = mozresource
+
+parsed = ''
+
+def generate_payload(server_data):
+    # TODO When HTMLLinkElement.sheet is available, don't need this
+    #      workaround anymore.
+    global parsed
+
+    params = urlparse.parse_qs(parsed.query)
+    stash_id = params['id'][0]
+
+    req = urllib2.Request('http://127.0.0.1:8000/_mozilla/mozilla/referrer-policy/generic/subresource/stash.py?id=%s' % stash_id)
```

Comment:
I tried the API a bit with no luck. I get empty response text in `common.js` after sending XMLHttpRequest, but calling `request.server.stash.take(request.GET["id"])` in `subresource/link.py` do give me the correct data.

Is `response.server.stash` shares with all subresource or did I use it the wrong way?